File: FrontEnd/src/models/workflowInstancesDTO.ts
export interface WorkflowInstancesDTO {
  id: string;
  createdDate: Date | null;
  status: number | null;
  workflowName: string | null;
  workflowCode: string | null;
  isCompleted: boolean;
  isTerminated: boolean;
  isDraft: boolean;
  defaultCompletedStatus: string | null;
  templateId: string;
  currentStep: number;
  unitId: string;
  itemId: string;
  createdBy: string;
  requestToChange: boolean;
  message: string | null;
  nameStatus: string | null;
  isApproved: boolean;
}

File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/WorkflowInstancesDto.cs
namespace DPD.HumanResources.Dtos.Dto;

public class WorkflowInstancesDto
{
    public Guid Id {get;set;}
    public DateTime? CreatedDate {get;set;}
    public int? Status {get;set;}
    public string WorkflowCode {get;set;}
    public string? WorkflowName {get;set;}
    public bool IsCompleted {get;set;}
    public bool IsTerminated {get;set;}
    public bool IsDraft {get;set;}
    public string? NameStatus {get;set;}
    public Guid TemplateId {get;set;}
    public int CurrentStep {get;set;}
    public Guid UnitId {get;set;}
    public Guid ItemId {get;set;}
    public Guid CreatedBy {get;set;}
    public bool? RequestToChange {get;set;}
    public string? Message {get;set;}
    public int? NumberForCode {get;set;}
    public bool IsApproved {get;set;}
}